plugins/ModuleLevelTrigger.cpp

```diff
@@ -148,10 +149,22 @@ ModuleLevelTrigger::create_decision(const triggeralgs::TriggerCandidate& tc)
   decision.trigger_number = m_last_trigger_number + 1;
   decision.run_number = m_run_number;
   decision.trigger_timestamp = tc.time_candidate;
-  // TODO: work out what to set this to
-  decision.trigger_type = 1; // m_trigger_type;
   decision.readout_type = dfmessages::ReadoutType::kLocalized;
 
+  // TODO: work out what to set this to
```

Comment:
I guess this comment is obsolete and can be removed

Reply:
This comment was there before my changes, specifically before the line:
decision.trigger_type = 1; // m_trigger_type;
This line is still there if the passthrough option is not set, so maybe we still want to change the assignment in the future (ie keep the comment, just move it to just before that line)? 
I have deleted it for now, but let me know what's better.

Reply:
I think it's fine for now, thanks